demos/ASP.Net MVC/Divergent.Frontend/Views/Orders/Details.cshtml

```diff
@@ -16,12 +15,3 @@
         <div>Courier: @Model.ShippingCourier</div>
     </div>
 </div>
-
-@*<div class="panel panel-default">
-    <div class="panel-heading">
-        <h3 class="panel-title">Shipment information</h3>
-    </div>
-    <div class="panel-body">
-
-    </div>
-</div>*@
```

Comment:
Having multiple demo solution should in the same repo has this side effect, not fun.
Which change/commit is "that" change?